### ISSUE TITLE
4FGL-DR3, KeyError: 'PLEC_Expfactor'

### ISSUE THREAD

**Issue by Sanjuphysics12** [fermiPy/fermipy#442]
If I use 4FGL-DR3 catalogue then I get the following error. Please, let me know the possible solution:

fermipy version v1.0.1 
ScienceTools version 2.0.8
Traceback (most recent call last):
  File "ATel.py", line 4, in <module>
    gta = GTAnalysis('config.yaml',logging={'verbosity' : 3}) #In the output we can see the version of `fermipy' and `fermi science tools'.
  File "/home/skmondal/anaconda3/envs/fermi/lib/python3.7/site-packages/fermipy/gtanalysis.py", line 351, in __init__
    coordsys=configs[0]['binning']['coordsys'])
  File "/home/skmondal/anaconda3/envs/fermi/lib/python3.7/site-packages/fermipy/roi_model.py", line 1746, in create
    config, **kwargs)
  File "/home/skmondal/anaconda3/envs/fermi/lib/python3.7/site-packages/fermipy/roi_model.py", line 1776, in create_from_source
    srcname=name, **kwargs)
  File "/home/skmondal/anaconda3/envs/fermi/lib/python3.7/site-packages/fermipy/roi_model.py", line 1352, in __init__
    self.load(coordsys=coordsys, srcname=srcname)
  File "/home/skmondal/anaconda3/envs/fermi/lib/python3.7/site-packages/fermipy/roi_model.py", line 1695, in load
    srcname=srcname)
  File "/home/skmondal/anaconda3/envs/fermi/lib/python3.7/site-packages/fermipy/roi_model.py", line 1975, in load_fits_catalog
    cat = catalog.Catalog.create(name)
  File "/home/skmondal/anaconda3/envs/fermi/lib/python3.7/site-packages/fermipy/catalog.py", line 168, in create
    return Catalog4FGL(fitsfile)
  File "/home/skmondal/anaconda3/envs/fermi/lib/python3.7/site-packages/fermipy/catalog.py", line 462, in __init__
    self._fill_params(self.table)
  File "/home/skmondal/anaconda3/envs/fermi/lib/python3.7/site-packages/fermipy/catalog.py", line 484, in _fill_params
    np.exp(tab['PLEC_Expfactor'][m] *
  File "/home/skmondal/anaconda3/envs/fermi/lib/python3.7/site-packages/astropy/table/table.py", line 1349, in __getitem__
    return self.columns[item]
  File "/home/skmondal/anaconda3/envs/fermi/lib/python3.7/site-packages/astropy/table/table.py", line 191, in __getitem__
    return OrderedDict.__getitem__(self, item)
KeyError: 'PLEC_Expfactor'

**Comment by ldivenere** [comment]
This issue should be solved when this pull request #441 . In the mean time, you can try installing directly from the new branch ( https://github.com/ldivenere/fermipy/tree/431-plsec4 ) to test fermipy with 4FGL-DR3.

**Comment by henrikef** [comment]
Hi @Sanjuphysics12  ! This should (I hope) be fixed now in the pre-release version (still being tested). If you want, you could test the development version of the fermitools and corresponding version of fermipy:

```
mamba create -n fermi-21 -c fermi -c conda-forge -c fermi/label/dev "fermitools>=2.1.0"
mamba activate fermi-21
pip install git+https://github.com/fermiPy/fermipy.git@v1.1.1
```

(I recommend using `mamba` instead of `conda`, it has a similar UI but resolves requirements much faster. You can try `conda` instead).

Please let me know if that helps.
